App.js

```diff
@@ -66,11 +69,14 @@ const driverStackNavigator = createStackNavigator({
   Rating: {
     screen: Rating
   },
+  AccountProfile: {
+    screen: AccountProfile
+  }
 });
 
 const TabNavigator = createBottomTabNavigator({
   ShopSearch: shopStackNavigator,
-  Notifications: Notifications,
+  Notifications: YourResults,
```

Comment:
Change this back to Notifications - always make sure you review your PR for simple mistakes like these!